custom_components/omnik_inverter/config_flow.py

```diff
@@ -141,6 +144,46 @@ async def async_step_setup_html(
             errors=errors,
         )
 
+    async def async_step_setup_tcp(
+        self, user_input: dict[str, Any] | None = None
+    ) -> FlowResult:
+        """Handle setup flow for tcp route."""
```

Comment:
```suggestion
        """Handle setup flow for TCP route."""
```

Reply:
I think I copied that from the `html` variant :)